docs/NSR.md

```diff
@@ -17,21 +17,21 @@ a.	Segment Certificate Systems into networks based on their functional or logica
 
 b.	Apply equivalent security controls to all systems co-located in the same network with a Certificate System;
 
-c.	Maintain Root CA Systems in a High Security Zone and in an offline state or air-gapped from all other networks;
+c.	Maintain Root CA Systems in a High Security Zone and in an offline state or air-gapped from all other networks. Offline Root CA systems shall be a non-networked, stand-alone system that is powered down when not in use; 
```

Comment:
I think some of these requirement can be clarified if definitions are provided to the terminology used:

Air-gapped: Physically segregated (NOTE: it is not sufficient for logical segregation alone (e.g., firewall rules, VLANs, etc.)

Offline: Stand-alone or networked computer system that is maintained in a powered off state when not actively in use.

Non-Networked - System that is only accessible  from a console, and cannot access any resources outside of those directly connected to the system.